clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2212,6 +2212,39 @@ device kernel, the attribute is ignored and it is not propagated to a kernel.
   }];
 }
 
+def SYCLIntelStallEnableAttrDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "intel::stall_enable";
+  let Content = [{
+Applies to a device function/lambda function. Requests, to the extent
```

Comment:
I would like this changed to: "When applied to a lambda or
function call operator (of a function object) on device, this requests, to the extent ..."

Reply:
Done